fhirpipe/parse/fhir.py

```diff
@@ -170,27 +182,33 @@ def bind_reference(fhir_object, fhir_spec):
     """
 
     # First we check that the reference has been provided
-    if fhir_object and fhir_object['identifier']['value']:
-        identifier = fhir_object['identifier']['value']
+    if fhir_object and fhir_object["identifier"]["value"]:
+        identifier = fhir_object["identifier"]["value"]
 
         # Collect all the resource_types which could be referenced
         try:
-            resource_types = re.search(
-                '\((.*)\)', fhir_spec["type"]).group(1).split('|')
+            resource_types = (
+                re.search("\((.*)\)", fhir_spec["type"]).group(1).split("|")
```

Comment:
Maybe we need to put the regexp out of the function to compile it a single time?
```
pattern = re.compile(r'([A-Z]+)([0-9]+)')
...
re.search(pattern, fhir_spec["type"]).group(1).split("|")
```

Reply:
yes, not sure it's very expensive but it cannot hurt. Done.